src/services/venue-service.ts

```diff
@@ -22,33 +12,40 @@ export interface GeoJsonFeature {
   };
 }
 
-export const emptyVenue = {
-  Id: 0,
-  name: 'Page Unavailable ',
+/**
+ * Default Venue values, to overwrite just set the property.
+ */
+const DEFAULT_VENUE: Venue = {
+  id: -1,
   address: '',
-  latitude: 0,
-  longitude: 0,
-  url: '',
+  name: 'Page Unavailable',
+  latitude: 39.758117, // default to skookum coordinates
+  longitude: -104.989888, // default to skookum coordinates
+  geoRadius: 50,
   companyImage: require('../images/company-images/emptyVenue.jpeg'),
-  description: '',
+  url: 'https://www.google.com/',
+  description: `
+  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque ac quam
+  lacinia, egestas libero vel, dictum eros. Etiam ut leo quis diam congue elementum
+  et non mi. Maecenas ut ante ut ante rhoncus tincidunt vitae sit amet quam.
+  `,
 };
-export class VenuesService  {
+
+export class VenuesService {
   public venues = [
     {
+      ...DEFAULT_VENUE,
```

Comment:
The spread operator really helps keep dummy data easily manageable.  I started with this just because I wanted to set a default `geoRadius`, then I realized we had a lot of shared data.  Even though the API will wipe this out soon enough....